moviepy/audio/fx/audio_delay.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+
+from moviepy.decorators import audio_video_fx
+from moviepy.audio.fx.multiply_volume import multiply_volume
+from moviepy.audio.AudioClip import CompositeAudioClip
+
+
+@audio_video_fx
+def audio_delay(clip, offset=0.2, n_repeats=8, decay=0.2):
+    """Return an audio (or video) clip whose audio is repeated certain number
+    of times with gaps between with a specific decay in its volume level.
+
+    Parameters
+    ----------
+
+    offset (float)
+      Gap between repetitions start times, in seconds.
+
+    n_repeats (int)
+      Number of repetitions (without including the clip itself).
+
+    decay (float)
+      Multiplication factor for the volume level of the last repetition. Each
+      repetition will have a value in the linear function between 1 and this value,
+      increasing or decreasing constantly. Keep in mind that the last repetition
+      will be muted if this is 0, and if is greater than 1, the volume will increase
+      for each repetition.
+
+    Examples
+    ========
```

Comment:
```suggestion
    offset : float, optional
      Gap between repetitions start times, in seconds.

    n_repeats : int, optional
      Number of repetitions (without including the clip itself).

    decay : float, optional
      Multiplication factor for the volume level of the last repetition. Each
      repetition will have a value in the linear function between 1 and this value,
      increasing or decreasing constantly. Keep in mind that the last repetition
      will be muted if this is 0, and if is greater than 1, the volume will increase
      for each repetition.

    Examples
    --------
```

According to https://numpydoc.readthedocs.io/en/latest/format.html#sections, this is the preferred formatting style.